packages/launchpad/src/setup-actions.ts

```diff
@@ -41,6 +42,11 @@ export const setUpActionsForTasks = ( {
 			}
 
 			action = () => {
+				if ( task.id === 'add_about_page' ) {
+					updateLaunchpadSettings( siteSlug as string, {
+						checklist_statuses: { add_about_page: true },
+					} );
+				}
```

Comment:
Might it be worth generalising this for future tasks that should be completed on click? I am thinking something like the following:
```suggestion
const TASKS_TO_COMPLETE_ON_CLICK = [ 'add_about_page' ]; // this could be anywhere in this file.
//...
				if ( siteSlug && TASKS_TO_COMPLETE_ON_CLICK.includes( task.id ) ) {
					updateLaunchpadSettings( siteSlug, {
						checklist_statuses: { [ task.id ]: true },
					} );
				}
```

Reply:
I think we have a few items where we're triggering completion from the target page, and it would be cleaner to have the logic in a more centralised location.

Reply:
I will create an issue for that too.